[SGLANG] Triton Attention UT failed

### Describe the bug

When testing the latest sglang on PVC, the triton attention kernel UT failed

```shell
git clone https://github.com/sgl-project/sglang.git
cd sglang
pip install -e "python[all_xpu]"
cd test/srt
pytest test_triton_attention_backend.py
```

The results will be 
```shell
FAILED test_triton_attention_kernels.py::TestTritonAttention::test_extend_attention - AssertionError: False is not true
FAILED test_triton_attention_kernels.py::TestTritonAttention::test_grouped_decode_attention - AssertionError: False is not true
======================================================================================================================= 2 failed, 2 passed in 195.69s (0:03:15) ========================================================================================================================
```


### Environment details

Triton: 3.2.0+git7bd27ed6
Torch: 2.7.0a0+git8326d27-cp310-cp310
GPU: 1550 with drivers 1032.21

@EikanWang @etaf @liangan1 for awareness

--- a/include/triton/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVMBase.h
+++ b/include/triton/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVMBase.h
@@ -1,6 +1,7 @@
 #ifndef TRITON_CONVERSION_TRITONGPU_TO_ELEMENTWISE_OP_H
 #define TRITON_CONVERSION_TRITONGPU_TO_ELEMENTWISE_OP_H
 
+#include "intel/include/Dialect/TritonIntelGPU/IR/Attributes.h"
 #include "mlir/Conversion/LLVMCommon/Pattern.h"
 #include "mlir/Conversion/LLVMCommon/TypeConverter.h"
 #include "mlir/Support/LLVM.h"
@@ -86,7 +87,8 @@ public:
       return resultVals;
     while (auto sliced = dyn_cast<SliceEncodingAttr>(baseEncoding))
       baseEncoding = sliced.getParent();
-    if (isa<LinearEncodingAttr, DotOperandEncodingAttr>(baseEncoding)) {
+    if (isa<LinearEncodingAttr, DotOperandEncodingAttr,
+            intel::DpasEncodingAttr>(baseEncoding)) {
       // TODO: this logic seems incorrect for mma layout. Skip for now.
       // The following test crashes and some other miscompile:
       // test_core::test_fp8_dot_acc

